Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
+
+class User{
+    public:
+    string Username;
+    string Email;
+    User(string username,string email){
```

Comment:
Use initializer lists in constructors for better performance and cleaner code. [medium]